apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -277,36 +346,83 @@ defmodule OMG.Watcher.ExitProcessor do
     {:reply, {:ok, db_updates ++ db_updates_from_state}, new_state}
   end
 
+  @doc """
+  See `piggyback_exits/1`. Flow:
+
+  - takes a list of IFE piggybacking events from the contract
+  - updates the `ExitProcessor`'s state
+  - returns `db_updates`
+  """
   def handle_call({:piggyback_exits, exits}, _from, state) do
     _ = if not Enum.empty?(exits), do: Logger.info("Recognized piggybacks: #{inspect(exits)}")
     {new_state, db_updates} = Core.new_piggybacks(state, exits)
     {:reply, {:ok, db_updates}, new_state}
   end
 
+  @doc """
+  See `challenge_exits/1`. Flow:
+
+  - takes a list of standard exit challenge events from the contract
+  - updates the `ExitProcessor`'s state
+  - returns `db_updates`
+  """
   def handle_call({:challenge_exits, exits}, _from, state) do
     _ = if not Enum.empty?(exits), do: Logger.info("Recognized challenges: #{inspect(exits)}")
     {new_state, db_updates} = Core.challenge_exits(state, exits)
     {:reply, {:ok, db_updates}, new_state}
   end
 
+  @doc """
+  See `new_ife_challenges/1`. Flow:
+
+  - takes a list of IFE exit canonicity challenge events from the contract
+  - updates the `ExitProcessor`'s state
+  - returns `db_updates`
+  """
   def handle_call({:new_ife_challenges, challenges}, _from, state) do
     _ = if not Enum.empty?(challenges), do: Logger.info("Recognized ife challenges: #{inspect(challenges)}")
     {new_state, db_updates} = Core.new_ife_challenges(state, challenges)
     {:reply, {:ok, db_updates}, new_state}
   end
 
+  @doc """
+  See `challenge_piggybacks/1`. Flow:
+
+  - takes a list of IFE piggyback challenge events from the contract
+  - updates the `ExitProcessor`'s state
+  - returns `db_updates`
+  """
+
   def handle_call({:challenge_piggybacks, challenges}, _from, state) do
     _ = if not Enum.empty?(challenges), do: Logger.info("Recognized piggyback challenges: #{inspect(challenges)}")
     {new_state, db_updates} = Core.challenge_piggybacks(state, challenges)
     {:reply, {:ok, db_updates}, new_state}
   end
 
+  @doc """
+  See `respond_to_in_flight_exits_challenges/1`. Flow:
+
+  - takes a list of IFE exit canonicity challenge response events from the contract
+  - updates the `ExitProcessor`'s state
+  - returns `db_updates`
+  """
   def handle_call({:respond_to_in_flight_exits_challenges, responds}, _from, state) do
     _ = if not Enum.empty?(responds), do: Logger.info("Recognized response to IFE challenge: #{inspect(responds)}")
     {new_state, db_updates} = Core.respond_to_in_flight_exits_challenges(state, responds)
     {:reply, {:ok, db_updates}, new_state}
   end
 
+  @doc """
+  See `finalize_in_flight_exits/1`. Flow:
+
+  - takes a list of in-flight exit finalization events from the contract
+  - pulls current information on IFE transaction inclusion
+  - discovers the `OMG.State`'s native key for the finalizing exits (`utxo_pos`)
+    (`Core.prepare_utxo_exits_for_in_flight_exit_finalizations/2`)
```

Comment:
This is additional to the L333, why? Shall you add there as well? Can you extend the context (is it reference to `see:`)